pkg/util/scrubber/yaml_scrubber.go

```diff
@@ -80,9 +80,27 @@ func walk(data *interface{}, callback scrubCallback) {
 	}
 }
 
+// IsEnc returns true is the string match the 'ENC[...]' format
+// Borrowed from comp/core/secrets/utils
+func IsEnc(str string) (bool, string) {
+	// trimming space and tabs
+	str = strings.Trim(str, " 	")
+	if strings.HasPrefix(str, "ENC[") && strings.HasSuffix(str, "]") {
+		return true, str[4 : len(str)-1]
+	}
+	return false, ""
+}
```

Comment:
We are not using the string returned, maybe we could consider using:
```suggestion
// IsEnc returns true is the string match the 'ENC[...]' format
// Borrowed from comp/core/secrets/utils
func IsEnc(str string) bool {
	// trimming space and tabs
	str = strings.Trim(str, " 	")
	if strings.HasPrefix(str, "ENC[") && strings.HasSuffix(str, "]") {
		return true
	}
	return false
}
```